packages/calcite-components/src/components/menu-item/menu-item.tsx

```diff
@@ -437,13 +437,26 @@ export class CalciteMenuItem implements LoadableComponent, T9nComponent, Localiz
     );
   }
 
+  renderHrefIcon(dir: Direction): VNode {
+    return (
+      <calcite-icon
+        class={CSS.hoverHrefIcon}
+        icon={dir === "rtl" ? "arrow-left" : "arrow-right"}
+        key={CSS.dropdownAction}
+        scale="s"
```

Comment:
Does the icon scale need to adjust based on the menu item's scale?

Reply:
Menu / Menu Item don't currently have scale, but when added down the road this would change.